infullmvp-kotlin-koin/src/main/java/com/infullmobile/android/infullmvp/MvpPresenter.kt

```diff
@@ -0,0 +1,33 @@
+package com.infullmobile.android.infullmvp
+
+import android.content.Intent
+import android.os.Bundle
+
+interface MvpPresenter<T> {
+
+    var view: T
+
+    fun onResume() {
+
+    }
+
+    fun onPageShow() {
```

Comment:
This looks like a `ViewPager` specific thing.

I see there is `setUserVisibleHint` override in a presenter. Is there a persistent case for this being in a default implementation? We've never encountered it.

To me this looks like a sporadic case when we can provide this implementation in screen specific classes.